airflow-core/src/airflow/ui/src/pages/DagRuns.tsx

```diff
@@ -289,7 +311,17 @@ export const DagRuns = () => {
             ))}
           </Select.Content>
         </Select.Root>
-      </Flex>
+
+        <Box flex="1" maxW="200px">
```

Comment:
```suggestion
        <Box>
```
I don't think this has any effect.